tests/integration/components/render-events-test.js

```diff
@@ -0,0 +1,127 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('tooltip-on-element', 'Integration | Component | event handlers', {
+  integration: true
+});
+
+
+function assertTargetHasLazyRenderEvents(assert, $target, eventType="hover") {
+  const eventsObject = $._data($target[0], 'events');
+
+  function assertEventExists(event) {
+    // for some reason mouseenter events are stored as mouseover in the eventsObject
+    // same for mouseleave -> mouse out
+    let eventObjectName = event;
+    if (event === 'mouseenter') {
+      eventObjectName = 'mouseover';
+    } else if (event === 'mouseleave') {
+      eventObjectName = 'mouseout';
+    }
+
+    assert.ok(eventsObject && eventsObject[eventObjectName],
+        `the eventsObject exists and should have ${eventObjectName} event`);
+
+    let eventHandler = eventsObject[eventObjectName][0];
+    assert.equal(eventHandler.origType, event,
+        `the eventHandler's origType property should equal ${event}`);
+
+    assert.ok(eventHandler.namespace.indexOf('target-lazy-render-wrapper') >= 0,
+        'the eventHandler\'s namespace property be unique to ember-tooltips');
+  }
+
+  function assertNumEventsExist(num) {
```

Comment:
I think the use of these helpers is self-explanatory but would like at least one line of comments to explain why and when these function should be used. This would enabled contributors to more easily understand and add to these files.